src/vs/editor/browser/controller/editContext/native/nativeEditContext.ts

```diff
@@ -310,6 +311,11 @@ export class NativeEditContext extends AbstractEditContext {
 		this._screenReaderSupport.onWillPaste();
 	}
 
+	public onWillCopy(): void {
+		this.logService.trace('NativeEditContext#onWillCopy');
+		this.logService.trace('NativeEditContext#isFocused : ', this.domNode.domNode === getActiveElement());
```

Comment:
Do you also want to log the current focused element, would that help to know what was focused when the editcontext was not or is that information not so useful ?